test/turbopack-dev-tests-manifest.json

```diff
@@ -1479,6 +1479,15 @@
     "flakey": [],
     "runtimeError": false
   },
+  "test/development/app-dir/root-not-found-missing-root-layout": {
```

Comment:
```suggestion
  "test/development/app-dir/root-not-found-missing-root-layout/index.test.ts": {
```

Reply:
My bad, will double check next time.

Reply:
Should this goto `failed` not `pending`?

Reply:
probably

Reply:
taking away too much time, my bad.